src/data/lovelace.ts

```diff
@@ -11,7 +12,7 @@ export interface LovelaceConfig {
 export interface LovelaceViewConfig {
   index?: number;
   title?: string;
-  badges?: string[];
+  badges?: LovelaceBadgeConfig[];
```

Comment:
We should be backwards compatible, so could also be a `string[]`

Reply:
```suggestion
  badges?: Array<string | LovelaceBadgeConfig>;
```